microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -17,7 +17,7 @@ To submit an extension to the Microsoft Edge Add-ons store, you must be register
 
 ## Before you begin  
 
-If you don't have an account or if you have an existing commercial account with Partner Center, create a new [Microsoft account (MSA)][WindowsCommunityEverythingAboutMicrosoftAccounts] to register with the Microsoft Edge program.  To create a Microsoft account \(Outlook/live/Hotmail\), complete the following actions.  
+If you don't have an account or if you have an existing commercial account with Partner Center, create a new [Microsoft account (MSA)][WindowsCommunityEverythingAboutMicrosoftAccounts] (or use an existing one) to register with the Microsoft Edge program.  To create a Microsoft account \(Outlook/live/Hotmail\), complete the following actions.  
```

Comment:
```suggestion
If you don't have a Microsoft account (MSA), or if you have an existing commercial account with Partner Center, create a new [Microsoft account (MSA)][WindowsCommunityEverythingAboutMicrosoftAccounts] (or use an existing one) to register with the Microsoft Edge program.  To create a Microsoft account \(Outlook/live/Hotmail\), complete the following actions.  
```

Reply:
Delete the phrase "or if you have an existing commercial account with Partner Center"?  The logic seems "off".  Do we want to say, "Even if you already have an existing commercial account with Partner Center, to register with the Microsoft Edge program, you will still need to use a Microsoft account (MSA)."

Reply:
Compare the logic of the wording in other PR, 
"Update aad-account.md"  https://github.com/MicrosoftDocs/edge-developer/pull/1392/files 
Seems to be spelled out better.